admin_guide/securing_builds.adoc

```diff
@@ -66,13 +66,41 @@ Default roles are provided which grant the *create* permission on the build stra
 == Disabling a Build Strategy Globally
 To prevent access to a particular build strategy globally, log in as a user with
 xref:../architecture/additional_concepts/authorization.adoc#roles[*cluster-admin*]
-privileges and remove the corresponding role from the *system:authenticated* group:
+privileges, remove the corresponding role from the *system:authenticated* group, and apply the annotation `openshift.io/reconcile-protect: "true"` to protect them from changes between the API restarts. The following example shows disabling the docker build strategy.
 
+. Apply `openshift.io/reconcile-protect` annotation
```

Comment:
s/`openshift.io/reconcile-protect` annotation/the `openshift.io/reconcile-protect` annotation: